src/i18n/de/nav.ts

```diff
@@ -1,94 +1,81 @@
 import { NavDictionary } from '../translation-checkers';
 
 export default NavDictionary({
-	// Start Here
-	startHere: 'Beginne hier',
+	welcomeWorld: 'Willkommen, Welt!',
 	'getting-started': 'Erste Schritte',
-	install: 'Installation',
-	'editor-setup': 'Editor-Einrichtung',
-	'upgrade-astro': 'Upgrade Astro',
+	'install-and-setup': 'Installation und Konfiguration',
+	'guides/deploy': 'Deine Website einrichten',
+	'guides/migrate-to-astro': 'Zu Astro migrieren',
+	'blog-tutorial': 'Tutorial: Baue einen Blog',
 
-	// Core Concepts
 	coreConcepts: 'Kernkonzepte',
-	'concepts/why-astro': 'Warum Astro',
+	'concepts/why-astro': 'Warum Astro?',
 	'concepts/islands': 'Astro-Inseln',
 
-	// Tutorials
-	tutorials: 'Tutorials',
-	'blog-tutorial': 'Baue einen Blog',
-	'add-collections-tutorial': 'Erweitere mit Content-Sammlungen',
-	'add-transitions-tutorial': 'Erweitere mit View-Transitions',
-
-	// Basics
 	basics: 'Grundlagen',
```

Comment:
```suggestion
	basics: 'Lerne die Grundlagen',
```
Die englische Seite hat hier inzwischen "Learn the Basics" stehen, daher würde ich vorschlagen, das auch in die Übersetzung zu übernehmen.